docs/en_US/TrainingService/KubeflowMode.md

```diff
@@ -198,6 +198,8 @@ Trial configuration in kubeflow mode have the following configuration keys:
   * image
     * Required key. In kubeflow mode, your trial program will be scheduled by Kubernetes to run in [Pod](https://kubernetes.io/docs/concepts/workloads/pods/pod/). This key is used to specify the Docker image used to create the pod where your trail program will run.
     * We already build a docker image [msranni/nni](https://hub.docker.com/r/msranni/nni/) on [Docker Hub](https://hub.docker.com/). It contains NNI python packages, Node modules and javascript artifact files required to start experiment, and all of NNI dependencies. The docker file used to build this image can be found at [here](https://github.com/Microsoft/nni/tree/master/deployment/docker/Dockerfile). You can either use this image directly in your config file, or build your own image based on it.
+  * privateRegistryFilePath
```

Comment:
how about `privateRegistryAuthPath` or `privateRegistryTokenPath`?

Reply:
changed to `privateRegistryAuthPath`